pkg/storage/client_split_test.go

```diff
@@ -2212,3 +2212,131 @@ func TestPushTxnQueueDependencyCycleWithRangeSplit(t *testing.T) {
 		})
 	}
 }
+
+func TestMinimal(t *testing.T) {
```

Comment:
ha, `TestMinimal` was something I use for the quick hacky test. Can/Should be renamed (also breaks my hacky workflows).